Products/ZenUI3/browser/resources/js/zenoss/eventclasses/ClassPanels.js

```diff
@@ -915,11 +916,19 @@ Ext.onReady(function(){
         setValue: function(value) {
             this.setTitle(_t("Transform for")+": "+value.transid);
             this.uid = "/zport/dmd"+value.transid;
-            this.down('codeeditorfield').setValue(value.trans);
+            str = this.setDefaultString(value.trans)
+            this.down('codeeditorfield').setValue(str);
         },
         reset: function() {
             this.down('codeeditorfield').setValue('');
+        },
+        setDefaultString: function(str){
```

Comment:
The name is misleading.  `setDefaultString` implies that you're setting the property named 'DefaultString'.
This function is actually getting a default string if the given string is empty.